### PR TITLE
Anton dev

### PR DESCRIPTION
**Pull request by Anton-Loginov**

### CODE REVIEW

File: src/assets/custom-template.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.page-template__custom-template {
+
+  .custom-template__box {
+    height: 330px;
```
Comment (MyronBatiuk) [inline src/assets/custom-template.scss]: Don't use fixed height here, you don't need it.

File: src/assets/footer-form.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.footer-form-template {
+.footer-form-template, .footer-form-template-help {
```
Comment (MyronBatiuk) [inline src/assets/footer-form.scss]: Use BEM structure ".footer-form__template". Fix this everywhere

File: src/assets/main.js [hunk 3]
```diff
@@ -53,8 +53,12 @@ $('.footer-form-btn').on('click', function() {
     localStorage.setItem('checkFreeTrial', true);
   }
 });
+
+$('.footer-form-template-help').hide();
+
 var checkUser = localStorage.getItem('checkFreeTrial');
 var checkUserLogin = localStorage.getItem('checkUserLogin');
```
Comment (MyronBatiuk) [inline src/assets/main.js]: Check if User is logged in liquid template, without JS.

File: src/sections/header.liquid [hunk 4]
```diff
@@ -202,7 +202,7 @@
                 {% if section.settings.align_logo == 'left' %}
                     {% if customer %}
                         {% assign customer_url = "/account" %}
```
Comment (MyronBatiuk) [inline src/sections/header.liquid]: Don't see any sense to create a variable , that is using only 1 time. You can add "/account" url right to the href.

File: src/sections/product-template.liquid [hunk 5]
```diff
@@ -210,10 +210,9 @@
                   </div>
               {% else %}
                   {% assign customer_url = "/account/login" %}
```
Comment (MyronBatiuk) [inline src/sections/product-template.liquid]: Same as comment above ^

File: src/assets/custom-template.scss (hunk 1 above)
Comment (Anton-Loginov) [reply src/assets/custom-template.scss]: Fixed.

File: src/assets/footer-form.scss (hunk 2 above)
Comment (Anton-Loginov) [reply src/assets/footer-form.scss]: Fixed.

File: src/sections/header.liquid (hunk 4 above)
Comment (Anton-Loginov) [reply src/sections/header.liquid]: Fixed.

File: src/sections/product-template.liquid (hunk 5 above)
Comment (Anton-Loginov) [reply src/sections/product-template.liquid]: Fixed.

**Pull request merged**